cli/viam/main.go

```diff
@@ -866,7 +865,7 @@ Example for linux/amd64:
 tar -czf packaged-module.tar.gz my-binary   # the meta.json entrypoint is relative to the root of the archive, so it should be "./my-binary"
 viam module upload --version "0.1.0" --platform "linux/amd64" packaged-module.tar.gz
                         `,
-						ArgsUsage: "<packaged-module.tar.gz>",
+						UsageText: "viam robot module upload <version> [other options] <package-module.tar.gz>",
```

Comment:
```suggestion
						UsageText: "viam module upload [options] <package-module.tar.gz>",
```

The module `upload` api is pretty clunky and we might want to change it before launching the registry.

Reply:
Updated to "viam module upload <version> <platform> [other options] <package-module.tar.gz>" (good catch on the extraneous "robot", but listing `<version>` and `<platform>` as required is probably good). As for the clunkiness of the API, this doesn't seem _too_ clunky to me on first glance, but I haven't actually used this API yet.

Reply:
internal docs still use `<packaged-module.tar.gz>`, in case you didn't intend to make this change.

Reply:
Oops switched back to `packaged`; thanks.